drastic-price-change-template/bot-config.json.example

```diff
@@ -0,0 +1,8 @@
+{
+  "priceDiscrepancyThreshold": 10,
```

Comment:
Not sure what the 10 represents? Is this for on-chain/off-chain discrepancies?

Reply:
Yes, it is